application/src/main/java/org/opentripplanner/osm/tagmapping/OsmTagMapper.java

```diff
@@ -59,6 +60,7 @@ public void populateProperties(WayPropertySet props) {
     props.setProperties("mtb:scale=4", noneWayProperties);
     props.setProperties("mtb:scale=5", noneWayProperties);
     props.setProperties("mtb:scale=6", noneWayProperties);
+    props.setProperties("highway=bridleway", withModes(NONE).bicycleSafety(1.3));
```

Comment:
https://wiki.openstreetmap.org/wiki/Tag:highway%3Dbridleway says that you're allowed to walk and cycle there.

Reply:
Well, in the UK at least but in Germany you're not...

Reply:
It is used when a bridleway is marked as `bicycle=yes`, which overrides the default permission.